test/visitor/var_usage.cpp

```diff
@@ -0,0 +1,93 @@
+/*************************************************************************
+ * Copyright (C) 2018-2019 Blue Brain Project
+ *
+ * This file is part of NMODL distributed under the terms of the GNU
+ * Lesser General Public License. See top-level LICENSE file for details.
+ *************************************************************************/
+
+#include "catch/catch.hpp"
+
+#include "parser/nmodl_driver.hpp"
+#include "test/utils/test_utils.hpp"
+#include "visitors/var_usage_visitor.hpp"
+
+using namespace nmodl;
+using namespace visitor;
+
+//=============================================================================
+// Variable usage visitor tests
+//=============================================================================
+
+bool run_var_usage_visitor(std::shared_ptr<ast::Node> node, const std::string& variable) {
+    return VarUsageVisitor().variable_used(node.get(), variable);
+}
+
+SCENARIO("Searching for variable name using VarUsageVisitor", "[visitor][var_usage]") {
+    auto to_ast = [](const std::string& text) {
+        parser::NmodlDriver driver;
+        return driver.parse_string(text);
+    };
+
+    GIVEN("A simple NMODL block") {
+        std::string nmodl_text = R"(
+            DERIVATIVE states {
+                tau = 11.1
+                exp(tau)
+                h' = h + 2 + n
```

Comment:
Maybe it would be useful to add an inner block in this test as well?
For example:
```suggestion
            {
                h' = h + 2 + n
            }